Update Readme.md (#10)
DevExpress-Examples/xaf-how-to-show-a-hyper-link-url-email-etc-for-a-business-class-property: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E2096)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# XAF - How to show a hyper link (URL, email, etc.) for a business class property

>  This example shows custom [Property Editors](https://docs.devexpress.com/eXpressAppFramework/113097/ui-construction/view-items-and-property-editors/property-editors) that provide access to object fields with email address or URL displayed as clickable text.

Reviewer: ```suggestion
 This example shows custom [Property Editors](https://docs.devexpress.com/eXpressAppFramework/113097/ui-construction/view-items-and-property-editors/property-editors) that provide access to object fields with an email address or URL as clickable text.
```

To validate a URL, these editors use a combined `RexEx` mask. The default regular expression is:

```
(((http|https|ftp)\://)?[a-zA-Z0-9\-\.]+\.[a-zA-Z]{2,3}(:[a-zA-Z0-9]*)?/?([a-zA-Z0-9\-\._\?\,\'/\\\+&amp;%\$#\=~])*)|([a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6})
```

You can use it as is or modify it per your specific needs. Search for [Regular Expression](https://learn.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-language-quick-reference) in MSDN for more information on how to do this.
A double click opens a URL in DetailView of the WinForms project for end-user convenience.

> These editors are created for learning purposes only. Feel free to extend them further or create your own editors to meet all your business needs.

Reviewer: ```suggestion
These editors are created for learning purposes only. You can extend them or create your own editors to meet your business needs.
```
